public/docs/_examples/toh-5/dart/lib/app_component.dart

```diff
@@ -0,0 +1,56 @@
+// #docplaster
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'package:angular2_tour_of_heroes/heroes_component.dart';
+import 'package:angular2_tour_of_heroes/hero_service.dart';
+import 'package:angular2/router.dart';
+import 'package:angular2_tour_of_heroes/dashboard_component.dart';
+// #docregion hero-detail-import
+import 'package:angular2_tour_of_heroes/hero_detail_component.dart';
+// #enddocregion hero-detail-import
+
+@Component(
+    selector: 'my-app',
+// #docregion template
+    template: '''
+<h1>{{title}}</h1>
+<nav>
+  <a [routerLink]="['Dashboard']">Dashboard</a>
+  <a [routerLink]="['Heroes']">Heroes</a>
+</nav>
+<router-outlet></router-outlet>
+''',
+// #enddocregion template
+// #docregion style-urls
+    styleUrls: const ['app_component.css'],
+// #enddocregion style-urls
+    directives: const [ROUTER_DIRECTIVES],
```

Comment:
Sorry, but I guess not. I have myself involved in too many things already. I need to be more productive on my own work.

Maybe this helps a bit

for TS it can be done this way (added to `bootstra(...)`
```ts
provide(PLATFORM_DIRECTIVES, {useValue: [ROUTER_DIRECTIVES], multi: true})
```

in Dart (see last line, rest just for context)

```dart
- angular2:
    entry_points:
    - web/main.dart
    platform_pipes: 'package:angular2/common.dart#COMMON_PIPES'
    platform_directives:
    - 'package:angular2/common.dart#COMMON_DIRECTIVES'
    - 'package:angular2/router.dart#ROUTER_DIRECTIVES'
```
this way the `ROUTER_DIRECTIVES` don't need to be added to each component